README.md

```diff
@@ -1,64 +1,49 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Module 1 - Capstone Project
 
-> One paragraph statement about the project.
+> Building my first Capstone project to apply the concepts learned in order to enhance my skills in HTML, CSS & JavaScript.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./imgs/Screenshot-desktop.png)
+
+![screenshot](./imgs/Screenshot-mobile.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
 - Major languages
-- Frameworks
-- Technologies used
+- HTML
+- JavaScript
+- CSS
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+[Live Demo Link]()
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+## Demo Video Link
 
+[Watch Demo](https://zairdon20.github.io/Last-capstone/)
```

Comment:
- [ ] Nice and descriptive readme👍. However, Your links are opening wrong pages, for example,[ live demo]() a link is opening 404 error Github page, and the [ Watch demo](https://zairdon20.github.io/Last-capstone/) a link is opening the live demo of your project 😊🍀

Reply:
thank you